LastWork(newWork)/Views/index/index.js

```diff
@@ -1,76 +1,134 @@
+        </div>
+        `;
+        interviewApp.EvaluationsPage.init();
+        const NAVnewEvalButton = document.querySelector('#newEvalButton');
+        NAVnewEvalButton.addEventListener('click', interviewApp.goNewEvalPag.init);
+        const logoutButton = document.querySelector('#logoutButton');
+        logoutButton.addEventListener('click', interviewApp.logout);
+        interviewApp.detailForm();
+    },
+    destroy() {
+        document.querySelector('#app').innerHTML = '';
+    },
 };
 
-const goNewEvalPag = function () {
-    document.querySelector('#app').innerHTML = NewEvaluationPage();
-    const NAVEvalButton = document.querySelector('#EvalButton');
-    NAVEvalButton.addEventListener('click', goEvalPag);
-    const logoutButton = document.querySelector('#logoutButton');
-    logoutButton.addEventListener('click', logout);
-    const submitButton = document.querySelector('#submitButton');
-    submitButton.addEventListener('click', aboutObject);
+interviewApp.goNewEvalPag = {
+    init() {
+        document.querySelector('#app').innerHTML =
+        `
+        <div id="newEvaluationPage">
+        </div>
+        `;
+        interviewApp.NewEvaluationPage.init();
+        const NAVEvalButton = document.querySelector('#EvalButton');
+        NAVEvalButton.addEventListener('click', interviewApp.goEvalPag.init);
+        const logoutButton = document.querySelector('#logoutButton');
+        logoutButton.addEventListener('click', interviewApp.logout);
+        const submitButton = document.querySelector('#submitButton');
+        submitButton.addEventListener('click', interviewApp.aboutObject);
+    },
+    destroy() {
+        document.querySelector('#app').innerHTML = '';
+    },
 };
 
-const formEvent = function (event) {
+interviewApp.formEvent = function (event) {
     event.stopPropagation();
     event.preventDefault();
 
     const username = document.querySelector('#username').value;
     const password = document.querySelector('#password').value;
 
-    if (checkUserDatas(username, password)) {
-        goEvalPag();
-    }
+    let logPromise = new Promise(((resolve, reject) => {
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', 'src/Data/loginDatas.json', true);
+        xhr.send();
+        xhr.onreadystatechange = function() {
+            if (this.readyState === 4) {
+                if (this.status < 400) {
+                    let loginDataResponse = xhr.responseText;
+                    try {
+                        loginDataResponse = JSON.parse(loginDataResponse);
+                        resolve(loginDataResponse);
+                    } catch (e) {
+                        reject('Cannot parse JSON');
+                    }
+                } else {
+                    reject('Cannot get JSON info');
+                }
+            }
+        }
+    }));
+
+    logPromise.then((loginData) => {
+        if (interviewApp.checkUserDatas(username, password, loginData)) {
```

Comment:
FYI / Off topic: `data` is already plural, meaning multiple pieces of information. The singular form is actually `datum`.